### DISCUSSION TITLE
More Control over Order of Sections

### DISCUSSION THREAD

**Discussion by puppyraeluna** [MicroPad/Web#442] [Ideas]
Is there a way to move the order of the sections and notes within the notebook to have things set in a desired order? It seems to be set up in chronological order of when it was created and not based on the user's style of organization. I do like having that chronological set up to start with but having the flexibility to move sections to be before or another another one without having to re-create the sections all over again in order to place in the user's desired order.

I'd also like to see that with the notes within the sections and possibly have content boxes that were created in one note to be able to be moved to another note, and maybe another one in another section, without having to re-create the content boxes from scratch.

**Comment by NickGeek** [comment]
Hi @puppyraeluna thanks for posting your suggestions.

> Is there a way to move the order of the sections and notes within the notebook to have things set in a desired order?

Sadly not right now, there's no drag-and-drop for sections/notes although that would be really nice to have. Currently the only main way to do this would be to edit the notepad file in a text editor. If you save your notepad to your device (Ctrl+S or Cmd+S for mac users) and then edit the *.npx file in an editor like Sublime Text or Visual Studio Code. It's quite janky and not fun, although with an XML formatting plugin (which will format the file to make it more human readable) it is doable.

> I'd also like to see that with the notes within the sections and possibly have content boxes that were created in one note to be able to be moved to another note, and maybe another one in another section, without having to re-create the content boxes from scratch.

That should be easily doable, I'll add that to my list.
